airflow-core/src/airflow/security/permissions.py

```diff
@@ -50,6 +50,7 @@
 RESOURCE_DOCS = "Documentation"
 RESOURCE_DOCS_MENU = "Docs"
 RESOURCE_IMPORT_ERROR = "ImportError"
+RESOURCE_REQUIRED_ACTIONS = "Required Actions"
```

Comment:
Maybe let's make it `RESOUCE_HITL_DETAILS`

Reply:
also related to https://github.com/apache/airflow/pull/54043/files, but we'll merge this UI one first